warp10/src/main/java/io/warp10/script/WarpScriptLib.java

```diff
@@ -447,192 +446,835 @@
+    addNamedWarpScriptFunction(new MANAGERONOFF(METAON, WarpManager.META_DISABLED, true));   
+    addNamedWarpScriptFunction(new MANAGERONOFF(METAOFF, WarpManager.META_DISABLED, false));   
+    addNamedWarpScriptFunction(new MANAGERONOFF(DELETEON, WarpManager.DELETE_DISABLED, true));   
+    addNamedWarpScriptFunction(new MANAGERONOFF(DELETEOFF, WarpManager.DELETE_DISABLED, false));
     
     addNamedWarpScriptFunction(new NOOP(BOOTSTRAP));
 
-    addNamedWarpScriptFunction(new RTFM("RTFM"));
-    addNamedWarpScriptFunction(new MAN("MAN"));
+    addNamedWarpScriptFunction(new RTFM(RTFM));
+    addNamedWarpScriptFunction(new MAN(MAN));
 
     //
     // Stack manipulation functions
     //
     
-    addNamedWarpScriptFunction(new PIGSCHEMA("PIGSCHEMA"));
+    addNamedWarpScriptFunction(new PIGSCHEMA(PIGSCHEMA));
     addNamedWarpScriptFunction(new MARK(MARK));
-    addNamedWarpScriptFunction(new CLEARTOMARK("CLEARTOMARK"));
-    addNamedWarpScriptFunction(new COUNTTOMARK("COUNTTOMARK"));
-    addNamedWarpScriptFunction(new AUTHENTICATE("AUTHENTICATE"));
-    addNamedWarpScriptFunction(new ISAUTHENTICATED("ISAUTHENTICATED"));
-    addNamedWarpScriptFunction(new STACKATTRIBUTE("STACKATTRIBUTE")); // NOT TO BE DOCUMENTED
-    addNamedWarpScriptFunction(new EXPORT("EXPORT"));
-    addNamedWarpScriptFunction(new TIMINGS("TIMINGS")); // NOT TO BE DOCUMENTED (YET)
-    addNamedWarpScriptFunction(new NOTIMINGS("NOTIMINGS")); // NOT TO BE DOCUMENTED (YET)
-    addNamedWarpScriptFunction(new ELAPSED("ELAPSED")); // NOT TO BE DOCUMENTED (YET)
-    addNamedWarpScriptFunction(new TIMED("TIMED"));
+    addNamedWarpScriptFunction(new CLEARTOMARK(CLEARTOMARK));
+    addNamedWarpScriptFunction(new COUNTTOMARK(COUNTTOMARK));
+    addNamedWarpScriptFunction(new AUTHENTICATE(AUTHENTICATE));
+    addNamedWarpScriptFunction(new ISAUTHENTICATED(ISAUTHENTICATED));
+    addNamedWarpScriptFunction(new STACKATTRIBUTE(STACKATTRIBUTE)); // NOT TO BE DOCUMENTED
+    addNamedWarpScriptFunction(new EXPORT(EXPORT));
+    addNamedWarpScriptFunction(new TIMINGS(TIMINGS)); // NOT TO BE DOCUMENTED (YET)
+    addNamedWarpScriptFunction(new NOTIMINGS(NOTIMINGS)); // NOT TO BE DOCUMENTED (YET)
+    addNamedWarpScriptFunction(new ELAPSED(ELAPSED)); // NOT TO BE DOCUMENTED (YET)
+    addNamedWarpScriptFunction(new TIMED(TIMED));
     addNamedWarpScriptFunction(new CHRONOSTART(CHRONOSTART));
     addNamedWarpScriptFunction(new CHRONOEND(CHRONOEND));
-    addNamedWarpScriptFunction(new CHRONOSTATS("CHRONOSTATS"));
-    addNamedWarpScriptFunction(new TOLIST("->LIST"));
-    addNamedWarpScriptFunction(new LISTTO("LIST->"));
-    addNamedWarpScriptFunction(new UNLIST("UNLIST"));
+    addNamedWarpScriptFunction(new CHRONOSTATS(CHRONOSTATS));
+    addNamedWarpScriptFunction(new TOLIST(TOLIST));
+    addNamedWarpScriptFunction(new LISTTO(LISTTO));
+    addNamedWarpScriptFunction(new UNLIST(UNLIST));
     addNamedWarpScriptFunction(new TOSET(TO_SET));
```

Comment:
Why use TO_SET instead of TOSET?

Reply:
That was already set as this. Should this PR also fix that ?